src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/add_panel/add_panel_flyout.test.tsx

```diff
@@ -80,45 +76,3 @@ test('createNewEmbeddable() add embeddable to container', async () => {
     lastName: 'bar',
   });
 });
-
-test('selecting embeddable in "Create new ..." list calls createNewEmbeddable()', async () => {
```

Comment:
This test makes no sense anymore and we have functional tests that are better suited to test interaction between components.